app/database/migration/server/index.ts

```diff
@@ -11,6 +11,17 @@ import {MM_TABLES} from '@constants/database';
 const {CHANNEL_BOOKMARK, CHANNEL_INFO, DRAFT, FILE, POST, CHANNEL, CUSTOM_PROFILE_ATTRIBUTE, CUSTOM_PROFILE_FIELD, SCHEDULED_POST} = MM_TABLES.SERVER;
 
 export default schemaMigrations({migrations: [
+    {
+        toVersion: 11,
+        steps: [
+            addColumns({
+                table: CHANNEL,
+                columns: [
+                    {name: 'policy_enforced', type: 'boolean', isOptional: true},
```

Comment:
Why optional? A boolean will default to false, I see no reason to keep ut optional.

Also can you explain what this is for those who have no context?

Reply:
>Why optional? A boolean will default to false, I see no reason to keep ut optional.

isOptional is on purpose, I can think on this two reasons:
- Older servers versions (or proxy caches) won’t include `policy_enforced` in the channel payload.
- not sure, but adding a non‑optional column would force a full DB reset or a heavy back‑fill migration.


> Also can you explain what this is for those who have no context?


This is part of the new feature Attribute‑Based Access Control (ABAC). So, in this feature an **access policy** is a defined and named set of rules that decide which users may join or stay in a private channel (this might change in the future and support other types) based solely on user attributes.

Reply:
> is a defined and named set of rules that decide which users may join or stay in a private channel 

And how is a boolean going to help me here?

If I have to guess, the boolean indicates that there is a policy but nothing more.

Also, how do I know this policy refers to abac and not to data retention, for me is not clear

Reply:
completely agree, will rename it to be called `abac_policy_enforced`

Reply:
Not renamed yet